### PR TITLE
[ENG-6191] Improve Styling and Add Total Users Label

### PR DESCRIPTION
**Pull request by Johnetordoff**
-   Ticket: [https://openscience.atlassian.net/browse/ENG-6191]
-   Feature flag: n/a

## Purpose

Standardize user tab look and make work for mobile, put in total users label.

## Summary of Changes

- standardize select colors
- add label for total number of users

## Screenshot(s)

Screenshot 2024-09-20 at 2 57 33 PM

### CODE REVIEW

File: app/institutions/dashboard/-components/institutional-users-list/template.hbs [hunk 1]
```diff
@@ -2,6 +2,12 @@
     <LoadingIndicator data-test-loading-indicator @dark={{true}} />
 {{else}}
     <div local-class='filter-container'>
+        <div local-class='total-users'>
+            <span local-class='total-users-count'>
+                {{@totalUsers}}
+            </span>
+            <label>{{t 'institutions.dashboard.users_list.total_users'}}</label>
```
Comment (futa-ikeda) [inline app/institutions/dashboard/-components/institutional-users-list/template.hbs:9]: `<label>` feels a bit inappropriate to use here, as there is no `<input>` that this is labelling. Could likely just be another `span` I think

File: app/institutions/dashboard/-components/institutional-users-list/styles.scss [hunk 2]
```diff
@@ -142,6 +142,11 @@
     margin-bottom: 10px;
 }
 
+.dropdown-trigger {
+    padding: 9px;
+    color: #337ab7;
```
Comment (futa-ikeda) [inline app/institutions/dashboard/-components/institutional-users-list/styles.scss:147]:
```suggestion
    color: $color-select;
```

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
Just a couple minor changes. Could you add a screenshot for mobile view to the PR as well?

**Review by futa-ikeda — APPROVED** [review]

**Comment by Johnetordoff** [comment]
@Johnetordoff remember to close the hide/add dropdown on clicking `apply`

File: app/institutions/dashboard/-components/total-count-kpi-wrapper/component-test.ts [hunk 3]
```diff
@@ -46,7 +46,7 @@ module('Integration | institutions | dashboard | -components | total-count-kpi-w
             .exists('The Project Widget exists');
 
         assert.dom('[data-test-total-count-kpi="1"]')
-            .hasText('20 OSF Public and Private Projects');
```
Comment (Johnetordoff) [inline app/institutions/dashboard/-components/total-count-kpi-wrapper/component-test.ts:49]: I'm not sure why this changed.

File: app/institutions/dashboard/-components/institutional-users-list/styles.scss [hunk 4]
```diff
@@ -247,3 +252,38 @@ input:checked + .slider::before {
...
+}
+
+.total-users-count {
+    font-size: 24px;
+    margin-right: 5px;
+    font-weight: bold;
+}
+
+/* Add media query for mobile devices */
+@media (max-width: 1000px) {
```
Comment (futa-ikeda) [inline app/institutions/dashboard/-components/institutional-users-list/styles.scss:275]: We've tried to move away from hard-coding pixel values into our CSS. Instead, in the template you could use our `is-mobile` helper
```
<div local-class=table-wrapper {{if (is-mobile 'mobile'}}>
```
then change this line to be `.mobile` and adjust the nested selectors as needed

File: app/institutions/dashboard/-components/institutional-users-list/styles.scss [hunk 5]
```diff
@@ -247,3 +252,38 @@ input:checked + .slider::before {
...
+    .table-wrapper {
+        overflow-x: auto;
+        -webkit-overflow-scrolling: touch; /* For smooth scrolling on iOS */
+    }
+
+    .table table {
+        min-width: 1000px; /* Set the table's width to be wider than the viewport */
```
Comment (futa-ikeda) [inline app/institutions/dashboard/-components/institutional-users-list/styles.scss:282]: I worry about setting an explicit `min-width` here, as the table may not need the full 1000px as users can hide columns and whatnot... Can this be implemented without this hard-coded min-width?

File: app/institutions/dashboard/-components/institutional-users-list/styles.scss (hunk 4 above)
Comment (futa-ikeda) [reply app/institutions/dashboard/-components/institutional-users-list/styles.scss:275]: Actually, I think this scrolling behavior and no-wrap on the text is something we may want regardless of mobile or not. Let me doublecheck with product though

File: app/institutions/dashboard/-components/institutional-users-list/template.hbs [hunk 6]
```diff
@@ -51,71 +57,73 @@
                         <Button @type='secondary' {{on 'click' dd.actions.close}}>
                             {{t 'general.cancel'}}
                         </Button>
-                        <Button @type='primary' {{on 'click' this.applyColumnSelection}}>
+                        <Button @type='primary' {{on 'click' this.applyColumnSelection}} {{on 'click' dd.actions.close}}>
```
Comment (futa-ikeda) [inline app/institutions/dashboard/-components/institutional-users-list/template.hbs:60]: Just for future reference, if you want to have a click even do multiple actions in sequence, you can use either the `pipe` or `queue` helper like 
`{{on 'click' (queue this.firstAction this.secondAction)}}`

Docs for those can be found here: https://github.com/DockYard/ember-composable-helpers?tab=readme-ov-file#queue

**Review by futa-ikeda — APPROVED** [review]
✨✨

**Pull request merged**